example/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { IntlProvider } from '@edx/frontend-i18n';
+
+import './index.scss';
+import Footer from '../src';
+
+const App = () => (
+  <div>
+    <IntlProvider locale="en">
```

Comment:
For what it's worth, you should be able to use frontend-base in this example app, like the header repos. (I don't feel strongly)

Reply:
I figure this is just as easy.